fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -59,11 +59,6 @@ def self.run(params)
           build_number = live_version.build.version
         end
 
-        # Remove leading zeros from version string (eg. 1.02 -> 1.2)
-        if version
-          version = version.split(".").map(&:to_i).join(".")
-        end
```

Comment:
@diskshima Any insights/thoughts on why this was done like this? If not, don't worry. I will dig into this later to review 😅